internal/server/server.go

```
@@ -261,36 +269,78 @@ func (s *Server) handleImport(writer http.ResponseWriter, request *http.Request)
+			break
+		}
+	}
+	if err := validateImportAccountID(accountID); err != nil {
+		return "", "", err
+	}
+	return token, accountID, nil
+}
+
+// parseImportToken preserves the original token-only helper for callers that
+// do not need to select a Team workspace.
+func parseImportToken(body []byte) (string, error) {
```

golangci-lint (2.12.2)

[error] func parseImportToken is unused

(unused)